code/src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -10,6 +10,8 @@
 
   <resource-files>
     <include path="/**.json" />
+    <include path="/**.p12" />
```

Comment:
I'm also not certain we want to check in the p12 file above. You usually don't check in binary files. What is it?

Reply:
yup we don't want to, it's the credentials access file for the service account